### DISCUSSION TITLE
RayTracing error

### DISCUSSION THREAD

**Discussion by makeclean** [idaholab/moose#17718] [Q&A General] [answered]
Hi, when running some of the ray tracing tests I get the following error
```
A 'LotsOfRaysRayStudy' is not a registered object.

If you are trying to find this object in a dynamically linked library, make sure that
the library can be found either in your "Problem/library_path" parameter or in the
MOOSE_LIBRARY_PATH environment variable
```
Just rebuilt MOOSE yesterday on commit b2d6c224b8536a88495889546a246301406148b5

**Comment by aeslaughter** [comment, marked as answer]
If you are running from the command-line, you probably just need to add `--allow-test-objects`

The objects is registered to the test app, which we disable by default to avoid test objects being used by accident.

https://github.com/idaholab/moose/blob/a2969bba1d652875edddc20414335a065518e5e1/modules/ray_tracing/test/src/userobjects/LotsOfRaysRayStudy.C#L18

**Comment by makeclean** [comment]
Thats it! Thanks
________________________________
